flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -1801,6 +1801,12 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
 
   @Test
   def testExtract(): Unit = {
```

Comment:
We don't need to add tests to the old test base. Can we remove this method entirely because `ExtractFunctionITCase` is much nicer.